modules/ROOT/pages/endpoints.adoc

```diff
@@ -0,0 +1,73 @@
+= Endpoints
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+An _endpoint_ is the entry point to a service, a process, or a queue or topic destination in service-oriented architecture. In APM, an endpoint defines the protocol, address, and other details specific to an exchange of messages between partners.
```

Comment:
don't use italics for emphasis. Italics is for variables.

Reply:
Why define endpoint but none of the other terms in this doc PR? Every developer who's ever accessed an API knows what an endpoint is. Don't use APM.

Reply:
Yes, this should have been source and target endpoint. My bad.